_posts/2022-04-08-mfa-phase-1-rollout.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Making popular Ruby packages more secure
+layout: post
+author: 
+author_email: 
+---
+
+Attacks on the software supply chain are increasing. Our community has not gone unscathed. RubyGems has been affected by supply chain attacks in the past. It’s important for us to mitigate these risks as much as possible. Recommending stronger security practices like enabling multi-factor authentication (MFA) on high-impact packages will be a first step in improving security of the RubyGems ecosystem.
```

Comment:
> Our community has not gone unscathed. RubyGems has been affected by supply chain attacks in the past.

I'm writing this with the best of intentions ❤️ . The rationale behind these sentences was to drive a compelling reason why RubyGems is doubling down on security. But I'm also cognizant that the Team may not want to call further attention to past incidents. If y'all want to drop this or replace it, I'm fully supportive of that ☺️.

Reply:
> Recommending stronger security practices like enabling multi-factor authentication (MFA) on high-impact packages will be a first step in improving security of the RubyGems ecosystem.

"will be" seems out of place. perhaps "is a first step"?